code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -33,14 +34,17 @@
 /obj/item/storage/box/large
 	name = "large box"
 	desc = "You could build a fort with this."
-	icon_state = "largebox"
+	icon_state = "large_box"
```

Comment:
was the original sprite for this ever cleaned up?

Reply:
Not sure what you mean, but yes, it's in the same file as all the boxes

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/69762909/c9cdebf6-8956-46d2-8dd6-2e84d02718a5)